specification/synapse/data-plane/Microsoft.Synapse/preview/2021-06-01-preview/runNotebook.json

```diff
@@ -0,0 +1,359 @@
+        },
+        "runStatus": {
+          "description": "Status of the run notebook.",
+          "type": "string"
+        },
+        "lastCheckedOn": {
+          "description": "Timestamp of last update.",
+          "type": "string"
+        },
+        "sessionId": {
+          "description": "Livy session id.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "sparkPool": {
+          "description": "SparkPool name.",
+          "type": "string"
+        },
+        "snapshot": {
+          "description": "Notebook output.",
+          "type": "object"
+        },
+        "sessionDetail": {
+          "description": "Run notebook session details.",
+          "type": "object"
+        },
+        "exitValue": {
+          "description": "Output of exit command.",
+          "type": "string"
+        },
+        "errors": {
+          "$ref": "#/definitions/Error"
+        }
+      }
+    },
+    "RunNotebookRequest": {
+      "description": "Run notebook request.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Run id (GUID)."
+        },
+        "notebook": {
+          "type": "string",
+          "description": "Notebook name."
+        },
+        "sparkPool": {
+          "type": "string",
+          "description": "SparkPool name."
+        },
+        "sessionOptions": {
+          "description": "Session properties.",
+          "$ref": "./entityTypes/Notebook.json#/definitions/NotebookSessionProperties"
+        },
+        "parameters": {
+          "type": "object",
+          "description": "Run notebook parameters",
+          "additionalProperties": {
+            "$ref": "#/definitions/parameter"
+          }
+        }
+      },
+      "required": [
+        "id",
+        "notebook"
+      ]
+    },
+    "parameter": {
+      "description": "Run notebook parameter.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Parameter type."
+        },
+        "value": {
+          "type": "object",
+          "description": "Parameter value."
+        }
+      }
+    },
+    "RunNotebookResponse": {
+      "description": "Run notebook response.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Http state code."
+        },
+        "message": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Response message."
+        },
+        "result": {
+          "$ref": "#/definitions/Result"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "VersionParameter": {
+      "name": "version",
+      "in": "path",
+      "maxLength": 50,
+      "required": true,
+      "type": "string",
+      "description": "Api version.",
+      "x-ms-parameter-location": "method"
```

Comment:
It's better to put it in client parameter. Unlike `RunId`, It won't change when using it:
```
"x-ms-parameter-location": "client"
```
Also we can give it a default value:
```
"x-ms-client-default": "2020-06-01-preview",
```